src/main/java/no/unit/nva/cristin/projects/FetchCristinProjects.java

```diff
@@ -128,10 +114,14 @@ private boolean isValidTitle(String str) {
 
     private boolean isValidCharacter(char c) {
         return Character.isWhitespace(c)
-                || Character.isLetterOrDigit(c)
-                || c == CHARACTER_DASH
-                || c == CHARACTER_COMMA
-                || c == CHARACTER_PERIOD;
+            || Character.isLetterOrDigit(c)
+            || c == CHARACTER_DASH
```

Comment:
These 3 chars could be a set of invalid characters and then you only  need to check INVALID_CHARS.contains(c)

Reply:
Those 3 chars are actually valid characters in addition to letter, digit and whitespace. But i can make a set of those character and do a VALID_CHARS.contains(c)

Reply:
Seems char array works a little different, so after discussing it with Orestis we decided to leave it as it is